### PR TITLE
Add video ads support

### PR DESCRIPTION
**Pull request by PawelPodkalicki**

### CODE REVIEW

File: tests/src/Supply/Domain/ValueObject/SizeTest.php [hunk 1]
```diff
@@ -43,4 +43,31 @@ public function testDimensions(): void
...
+        $this->assertContains('300x250', Size::findBestFit(300, 250, 1));
+        $this->assertContains('336x280', Size::findBestFit(330, 270, 1));
+    }
+
+    public function testFindMatching(): void
+    {
+        $this->assertEmpty(Size::findMatching(1, 1));
+        $this->assertEmpty(Size::findMatching(300, 0));
+        $this->assertEmpty(Size::findMatching(300, 10));
+        $this->assertNotEmpty(Size::findMatching(300, 250));
```
Comment (m-pilarczyk) [inline tests/src/Supply/Domain/ValueObject/SizeTest.php]: Check if array contains specific size

File: tests/app/Client/Mapper/AdSelect/CampaignMapperTest.php [hunk 2]
```diff
@@ -124,6 +124,29 @@ public function testMappingCampaignWithClassification(): void
...
+                        ]
+                    ),
+                ],
+            ]
+        );
+        $campaign = CampaignFactory::createFromArray($campaignData);
+
+        $mapped = CampaignMapper::map($campaign);
+
+        self::assertIsArray($mapped['banners'][0]['banner_size']);
```
Comment (m-pilarczyk) [inline tests/app/Client/Mapper/AdSelect/CampaignMapperTest.php]: Check returning value

File: app/Http/Controllers/Manager/CampaignsController.php [hunk 3]
```diff
@@ -57,13 +61,15 @@
 use Illuminate\Support\Facades\Mail;
 use Illuminate\Support\Facades\Response as ResponseFacade;
 use Illuminate\Support\Facades\Validator;
-use InvalidArgumentException;
-use RuntimeException;
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 use Symfony\Component\HttpKernel\Exception\UnprocessableEntityHttpException;
 
+use function array_map;
```
Comment (m-pilarczyk) [inline app/Http/Controllers/Manager/CampaignsController.php]: To delete

**Review by m-pilarczyk — CHANGES_REQUESTED** [review]

File: tests/src/Supply/Domain/ValueObject/SizeTest.php (hunk 1 above)
Comment (PawelPodkalicki) [reply tests/src/Supply/Domain/ValueObject/SizeTest.php]: OK

File: tests/app/Client/Mapper/AdSelect/CampaignMapperTest.php (hunk 2 above)
Comment (PawelPodkalicki) [reply tests/app/Client/Mapper/AdSelect/CampaignMapperTest.php]: OK

File: app/Http/Controllers/Manager/CampaignsController.php (hunk 3 above)
Comment (PawelPodkalicki) [reply app/Http/Controllers/Manager/CampaignsController.php]: OK

**Review by m-pilarczyk — DISMISSED** [review]

**Review by m-pilarczyk — APPROVED** [review]

**Pull request merged**